src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -63,12 +64,14 @@ public RepoConfiguration(RepoLocation location, String branch, List<Format> form
 
         String organization = location.getOrganization();
         String repoName = location.getRepoName();
+        displayName = repoName + "[" + branch + "]";
+        outputFolderName = repoName + "_" + branch;
 
         if (organization != null) {
-            displayName = organization + "_" + repoName + "_" + branch;
             repoFolderName = organization + "_" + repoName;
+            displayName = organization + "/" + displayName;
+            outputFolderName = organization + outputFolderName;
```

Comment:
What I want is:

```
repoFolderName = repoName;
displayName = repoName + "[" + branch + "]";
outputFolderName = repoName + "_" + branch;

if (organization != null) {
    repoFolderName = organization + "_" + repoFolderName ;
    displayName = organization + "/" + displayName;
    outputFolderName = organization + outputFolderName;
}
```

Is it more readable? 🤔

Reply:
It would be. But `repoFolderName` is final, so we can't prepend `organization` to it.

Reply:
Hmm, I don't think its that critical to use `final` for `repoFolderName` since we only use it through its getter. 🤔 

@eugenepeh, Should we remove the `final` keyword for better code readability?

Reply:
looks like its resolved